sim/build.sbt

```diff
@@ -87,6 +101,8 @@ lazy val firesim    = (project in file("."))
     Compile / doc := (doc in ScalaUnidoc).value,
     // Registers the unidoc-generated html with sbt-site
     addMappingsToSiteDir(mappings in (ScalaUnidoc, packageDoc), siteSubdirName in ScalaUnidoc),
-    concurrentRestrictions += Tags.limit(Tags.Test, 1)
+    concurrentRestrictions += Tags.limit(Tags.Test, 1),
+    mainClass in assembly := Some("chipyard.Generator"),
```

Comment:
This just specifies a _default_ main class in the jar right?

Reply:
Yes.  I have a suspicion that if you have more than one main class and you don't specify a default that assembly wigs out. I didn't test that theory, I just saw other projects were specifying a default and also did that.

Reply:
It's not strictly necessary to specify a `mainClass`, FIRRTL doesn't: https://github.com/freechipsproject/firrtl/blob/d1c0181e716c37142e233beed2efcea5c5794aa7/build.sbt#L65

But it is just the default.